web/xfa_layer_builder.css

```diff
@@ -13,6 +13,10 @@
  * limitations under the License.
  */
 
+.xfaPrintOnly {
+  display: none;
```

Comment:
Shouldn't these elements be displayed when the user wants to print the PDF?

Reply:
I need to check if it's relevant or not in general. For example in the pdf in the above bug, I'm not sure if it's correct correct to print these elements when maybe they should be dynamically modified.
So for now we just print what the user see.

Reply:
OK, can you file an issue about this so we don't forget?